joomla/joomla-cms
# [4.0] [plg_system_httpheaders] Drop option "Write headers to the configuration file"

Pull Request for Issue #25716

### Summary of Changes

As discussed here #25716 this here is removing the htaccess / web.config option from the plugin again for the reasons explained in that issue.

### Testing Instructions

Make sure the option is gone and nothing writes to the htaccess and web.config files.
make sure headers are still set correctly.

### Expected result

nothing writes to the htaccess and web.config files.

### Actual result

this is a option to write to the htaccess and web.config files.

### Documentation Changes Required

Yes.
